### ISSUE TITLE
Movies with multiple versions (but the same resolution) are sorted incorrectly

### ISSUE THREAD

**Issue by ashanrath** [jellyfin/jellyfin#12861]
### This issue respects the following points:

- [X] This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
- [X] This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aopen+is%3Aissue) _(I've searched it)_.
- [X] I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
- [X] I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
- [X] This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.

### Description of the bug

When adding multiple versions of a movie, they are sorted incorrectly for selection in the "Version" drop down list on the movie item page.

### Reproduction steps

Add movies with a similar naming pattern to a single folder, e.g.
- Star Wars Episode IV - A New Hope (1977) - Blu-Ray Release [Bluray-1080p][7.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Blu-Ray Release [Bluray-2160p][7.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Project 4K77 - v1.4 [Bluray-1080p][5.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Project 4K77 - v1.4 [Bluray-2160p][5.1].mkv

### What is the current _bug_ behavior?

The files are sorted to prefer higher resolution (expected) but are ignoring alphabetical order for files with the same resolution.
- Star Wars Episode IV - A New Hope (1977) - Project 4K77 - v1.4 [Bluray-2160p][5.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Blu-Ray Release [Bluray-2160p][7.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Project 4K77 - v1.4 [Bluray-1080p][5.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Blu-Ray Release [Bluray-1080p][7.1].mkv

### What is the expected _correct_ behavior?

As per the [documentation](https://jellyfin.org/docs/general/server/media/movies/), files should be sorted by resolution, then alphabetically. Expected order would therefor be:
- Star Wars Episode IV - A New Hope (1977) - Blu-Ray Release [Bluray-2160p][7.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Project 4K77 - v1.4 [Bluray-2160p][5.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Blu-Ray Release [Bluray-1080p][7.1].mkv
- Star Wars Episode IV - A New Hope (1977) - Project 4K77 - v1.4 [Bluray-1080p][5.1].mkv

### Jellyfin Server version

10.9.11+

### Specify the build version

10.9.11

### Environment

```markdown
- OS: Windows 11
- Linux Kernel: none
- Virtualization: none
- Clients: Browser, Tizen, Android, iOS
- Browser: Chrome, Edge
- FFmpeg Version: N/A
- Playback Method: Direct play, transcode
- Hardware Acceleration: Intel QSV
- GPU Model: Intel Iris Xe, 11th gen.
- Plugins: N/A
- Reverse Proxy: Traefik
- Base URL: /jellyfin
- Networking: host
- Storage: local
```

### Jellyfin logs

```shell
N/A
```

**Comment by TheMelmacian** [comment]
Known bug and already fixed:
https://github.com/jellyfin/jellyfin/issues/12618
https://github.com/jellyfin/jellyfin/pull/12621

Will probably be included in 10.10

**Comment by ashanrath** [comment]
Closing as duplicate of #12618.
